src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -764,7 +795,11 @@ namespace HVACSingleDuctInduc {
 				}
 			} else {
 				CheckZoneSizing( IndUnit( IUNum ).UnitType, IndUnit( IUNum ).Name );
-				MaxTotAirVolFlowDes = max( TermUnitFinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow );
+				if ( CurTermUnitSizingNum > 0 ) {
```

Comment:
Then MaxTotAirVolFlowDes always = max( TermUnitFinalZoneSizing( blah blah blah... regardless if CurTermUnitSizingNum > 0. So this is just a check to prevent an array access crash when CurTermUnitSizingNum = 0. OK then. The variable name confused me since it's not a ptr to the design spec object (which I assumed), it's an index to a TU in a list that is the number of total zone inlets nodes long. As long as the TermUnitFinalZoneSizing array is in that same order then it's OK. I guess if these weren't in the same order there would be massive diff's on the dashboard.